mypy/build.py

```diff
@@ -283,7 +283,7 @@ def default_lib_path(data_dir: str,
         # We allow a module for e.g. version 3.5 to be in 3.4/. The assumption
         # is that a module added with 3.4 will still be present in Python 3.5.
         versions = ["%d.%d" % (pyversion[0], minor)
-                    for minor in reversed(range(pyversion[1] + 1))]
+                    for minor in reversed(range(3, pyversion[1] + 1))]
```

Comment:
Oh, sorry I must have misunderstood, I thought you wanted that as a separate change. I'll make that change now as part of this.